### PR TITLE
show Alert For mismatched fileType

### PR DESCRIPTION
**Pull request by Tahia50**

### CODE REVIEW

**Comment by hasanmainul** [comment]
👍

File: FileReaderApp/ViewController.m [hunk 1]
```diff
@@ -69,4 +72,21 @@ - (void)setUpDropDownMenu {
     [self.view addSubview:self.menu];
 }
 
+- (IBAction)showPDF:(id)sender {
+    [self matchExtension:@"pdf"];
```
Comment (jannatulabeda) [inline FileReaderApp/ViewController.m]: "pdf" : Don't use hardcoded value

File: FileReaderApp/ViewController.m [hunk 2]
```diff
@@ -69,4 +72,21 @@ - (void)setUpDropDownMenu {
...
+}
+
+- (IBAction)showPPT:(id)sender {
+    [self matchExtension:@"ppt"];
```
Comment (jannatulabeda) [inline FileReaderApp/ViewController.m]: "ppt" : Don't use hardcoded value

File: FileReaderApp/FileAlert.m [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+//  Created by Tahia Ata on 5/13/16.
+//  Copyright © 2016 Tahia Ata. All rights reserved.
+//
+
+#import "FileAlert.h"
+#define kDefaultError  @"Mismatch Error"
+
+@implementation FileAlert
+
+
```
Comment (dina28) [inline FileReaderApp/FileAlert.m]: remove blank line

**Comment by dina28** [comment]
@Tahia50 👍

**Comment by jannatulabeda** [comment]
wi4ce

**Pull request merged**